client/mm/libxc/binance.go

```diff
@@ -338,23 +338,19 @@ func (b *binanceOrderBook) midGap() uint64 {
 
 // TODO: check all symbols
 var dexToBinanceSymbol = map[string]string{
-	"POLYGON": "MATIC",
-	"WETH":    "ETH",
+	"polygon": "MATIC",
+	"weth":    "ETH",
 }
 
 var binanceToDexSymbol = make(map[string]string)
 
-func convertBnCoin(coin string) string {
+// convertBnCoin converts a binance coin symbol to a dex symbol.
+func convertBnCoin(coin string, weth bool) string {
```

Comment:
I'm scratching my head a little at the meaning of the `weth bool` argument.

Reply:
Added this comment: 

```
// On Binance there is no distinction between "eth" and "weth".
// If weth is true, this function will return "weth", otherwise
// it will return "eth".
```

Reply:
Updated it.